### DISCUSSION TITLE
Поддержка устройств ARM64 и x86.

### DISCUSSION THREAD

**Discussion by iMallpa** [localsend/localsend#583] [L-Русский]
Как следует из заголовка, у меня есть компьютер с процессором архитектуры ARM. Мой друг настоятельно рекомендовал мне попробовать его, когда я узнал, что мне нужно программное обеспечение для переноса между устройствами. Но когда я загружаю файл .msix и запускаю его, Windows показывает, что мое устройство не поддерживается. Я очень разочарован
Мне очень нравится локальная отправка, и я надеюсь, что она сможет адаптироваться к большему количеству платформ и предоставлять высококачественные услуги пользователям большего количества платформ.
благодарный

**Comment by Drsheppard01** [comment]
Привет @iMallpa, приложение для Windows ARM64, вероятно, скоро появится: #715
Также вы можете попробовать установить Linux на базе Debian и попробовать Localsend на новой системе
С уважением, Dr. Sheppard (pingvinus.ru/user/drsheppard)
